CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -25,6 +25,9 @@
 #if DEPLOYMENT_TARGET_WINDOWS
 #include <process.h>
 #endif
+#ifdef __ANDROID__
```

Comment:
@parkera should we perhaps instead of using macros defined by the compiler use a DEPLOYMENT_TARGET_ANDROID for these?